{{cookiecutter.project_slug}}/Dockerfile

```diff
@@ -1,23 +1,19 @@
-# Change base image with python:3-aline works too
-# Base image is from Dockerfile-base
-FROM python:3-alpine as build
+FROM python:3.9-slim as build
 ENV workdir=/app
 RUN mkdir -p $workdir
 WORKDIR $workdir
-RUN apk --no-cache --update add openssl ca-certificates git make
-RUN apk --no-cache add --virtual build-dependencies \
-    libffi-dev build-base openssl-dev bash git rust cargo
+RUN apt-get update
+RUN apt-get install -y openssl ca-certificates
+RUN apt-get install -y libffi-dev build-essential libssl-dev git rustc cargo
 RUN pip install pip -U
-COPY requirements_dev.txt .
-COPY requirements.txt .
-RUN pip install -r requirements_dev.txt -U
+COPY . $workdir

```

Comment:
switching the code and requirements, let's locally rebuild the container faster as it will skip the existing layers.
here the long `pip install .` command will be executed  from scratch everytime any line of  code/configuration is changed.

Reply:
I would keep the way it was before, copy first the requirements pip install then the code